src/components/SmallCard.js

```diff
@@ -0,0 +1,36 @@
+import React,  {Component} from 'react';
+import {
+  Box,
+  Text
+} from 'grommet'
+
+
+
+export default class SmallCard extends Component{
```

Comment:
We are trying to stay away from writing classes and we are using functional components instead.